eslint.config.mjs

```
@@ -0,0 +1,48 @@
+import eslintjs from "@eslint/js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: eslintjs.configs.recommended,
+});
+
+export default [
+    {
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)